src/components/app/app.test.ts

```diff
@@ -189,7 +189,9 @@ describe('app test suite', () => {
     it('should store a session in a signed cookie', async () => {
       const response = await request(app).get('/test');
 
-      expect(response.header['set-cookie'][0]).toContain('pazmin-session.sig');
+      expect(response.header['set-cookie'].some(
+        (x: string) => x.indexOf('pazmin-session.sig') > -1))
```

Comment:
This is fine, but you could use: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/includes